jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# Sitespeed job script
+# Assumes the following parameters set
+# as environment variables
+#
+# TEST_URL
+# EDX_USER
+# EDX_PASS
+# USE_BASIC_AUTH
+# AUTH_USER (optional, required if USE_BASIC_AUTH)
+# AUTH_PASS (optional, required if USE_BASIC_AUTH)
+# NUMBER_OF_TIMES
+# CONNECTION
+
+echo "Cleaning out old results if they exist"
+rm -rf sitespeed-result
+mkdir -p sitespeed-result
+
+# create a file with the session credential cookie
+ARGS="edx-sitespeed/edx_sitespeed/edx_sitespeed.py -e ${EDX_USER} -p ${EDX_PASS} -u ${TEST_URL}"
+if [ -n "$USE_BASIC_AUTH" ] ;
+    then ARGS="${ARGS} --auth_user ${AUTH_USER} --auth_pass ${AUTH_PASS}"
```

Comment:
so, in this example, the end-result will pass `-p`, `-e`, as well as `--auth_user` and `--auth_pass`?